vividus-plugin-lighthouse/src/main/java/org/vividus/lighthouse/LighthouseSteps.java

```diff
@@ -360,8 +415,10 @@ private LighthouseResultV5 performLocalLighthouseScan(Path outputPath, String we
 
         try (InputStream stream = Files.newInputStream(outputPath))
         {
-            return pagespeedInsights.getObjectParser().parseAndClose(stream, StandardCharsets.UTF_8,
-                    LighthouseResultV5.class);
+            LighthouseResultV5 result = pagespeedInsights.getObjectParser().parseAndClose(stream,
+                    StandardCharsets.UTF_8, LighthouseResultV5.class);
+            result.put(FS_RESULT_PATH, outputPath.toString());
```

Comment:
it's not really correct to change the resulting object:
- it breaks consistency
- object in memory knows its location on the filesystem

it's better to change the returned result tyoe to entry: `Entry<Path, LighthouseResultV5>`